README.rst

```diff
@@ -0,0 +1,113 @@
+.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
+   :target: https://travis-ci.org/diffpy/diffpy.pdfmorph
+
+.. image:: http://codecov.io/github/diffpy/diffpy.pdfmorph/coverage.svg?branch=master
+   :target: http://codecov.io/github/diffpy/diffpy.pdfmorph?branch=master
+
+
+PDFmorph
+========================================================================
+# FIXME
+add description, sample below
+Code package for morphing atomic pair distribution functions.
+
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, 
+https://github.com/diffpy/diffpy.utils
+
+# FIXME - add link for instructions
+If you can't use conda and need to build from the sources, instructions are [here].
+
+INSTALLATION
+------------------------------------------------------------------------
+
+The preferred method is to use Anaconda Python and install from the
+"diffpy" channel of Anaconda packages ::
+
+     conda config --add channels diffpy
+     conda install diffpy.pdfmorph
+
+# FIXME
+PDFmorph can then be started from a terminal...
+[insert more instructions here]
+
+With Anaconda, PDFmorph can be later upgraded to the latest released
+version using ::
+
+     conda update diffpy.pdfmorph
+
+With other Python distributions the program can be upgraded to
+the latest version as follows ::
+
+     easy_install --upgrade diffpy.pdfmorph
+
+
+DEVELOPMENT
+------------------------------------------------------------------------
+
+PDFmorph is an open-source software available in a git repository at
+https://github.com/diffpy/diffpy.pdfmorph.
+
+Feel free to fork the project and contribute! To install PDFmorph
+in a development mode where the source files are used directly
+rather than copied to a system directory, use ::
+
+     python setup.py develop --user
+
+
+CONTACTS
+------------------------------------------------------------------------
+
+For more information on PDFmorph, visit the project wep-page:
+
+# FIXME - make a page for the project
+[insert project page here, when it exists]
+
+or email Professor Simon Billinge at sb2896@columbia.edu
+
+
+LICENSE
+------------------------------------------------------------------------
+This program is part of the DiffPy and DANSE open-source projects at Columbia
+University and is available subject to the conditions and terms laid out below.
+
+Copyright © 2009-2010, Trustees of Columbia University in the City of New York,
+all rights reserved.
+
+For more information please visit the diffpy web-page at http://diffpy.org or
+email Prof. Simon Billinge at sb2896@columbia.edu.
+
```

Comment:
@sbillinge Would eliminating the info below this line be an alright idea or is it all necessary to include in the readme? Could we introduce a different file with the rest of the licensing info?

Reply:
please see my comment below.